app/services/katello/candlepin/consumer.rb

```diff
@@ -0,0 +1,29 @@
+module Katello
+  module Candlepin
+    class Consumer
+      include LazyAccessor
+
+      attr_accessor :uuid
+
+      def initialize(uuid)
+        self.uuid = uuid
+      end
+
+      def regenerate_identity_certificates
+        Rails.logger.debug "Regenerating consumer identity certificates: #{name}"
+        Resources::Candlepin::Consumer.regenerate_identity_certificates(self.uuid)
+      rescue => e
+        Rails.logger.debug e.backtrace.join("\n\t")
+        raise e
+      end
+
+      def backend_data
+        Resources::Candlepin::Consumer.get(uuid)
```

Comment:
I don't know that that is a major concern, and it prevents you from overriding the getter later on if that was ever needed (without updating every @attribute reference).  I'm pretty sure we've had this discussion before :)